MAVProxy/mavproxy.py

```diff
@@ -1516,6 +1517,9 @@ def quit_handler(signum = None, frame = None):
     if opts.map:
         process_stdin('module load map')
 
+    if opts.mavcesium:
+        process_stdin('module load cesium')
```

Comment:
I'm not a great fan of this.  We have `--load-module` already.

Reply:
Ah, ok, I was just trying to make it like `mavproxy --map` but for 3D. I'll drop this.

Reply:
Dropped.